configs/selfsup/beit/beit_vit-base-p16_8xb256-amp-coslr-300e_in1k.py

```diff
@@ -0,0 +1,53 @@
+_base_ = [
+    '../_base_/models/beit_vit-base-p16.py',
+    '../_base_/datasets/imagenet_beit.py',
+    '../_base_/schedules/adamw_coslr-300e_in1k.py',
+    '../_base_/default_runtime.py',
+]
+
+# optimizer wrapper
+optimizer = dict(
+    type='AdamW', lr=1.5e-3, betas=(0.9, 0.999), weight_decay=0.05)
+
+optim_wrapper = dict(
+    type='AmpOptimWrapper',
+    loss_scale='dynamic',
+    optimizer=optimizer,
+    clip_grad=dict(max_norm=3.0),
+    paramwise_cfg=dict(
+        custom_keys={
```

Comment:
May add some comments to indicate the following configurations are designed for  BEiT